src/main/resources/db/migration/V2_0__create_vedlegg_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE vedlegg
+(
+    id         SERIAL PRIMARY KEY,
+    melding_id INTEGER REFERENCES melding (id) ON DELETE CASCADE,
+    uuid       VARCHAR(50) NOT NULL UNIQUE,
+    created_at TIMESTAMPTZ NOT NULL,
+    updated_at TIMESTAMPTZ NOT NULL,
+    number     INTEGER NOT NULL,
```

Comment:
Så hvert vedlegg får et tall 1,...,n for antall vedlegg per melding?

Reply:
Ikke helt. Tanken er å bevare rekkefølgen på vedleggene, så number=0 blir det første, number=1 det andre og så videre.

Reply:
Ja, ser jeg formulerte meg litt rart, men vi mente det samme 👍🏼 🚀 Men at den er null-indeksert i stedet, da.